src/pages/PointOfSale/Billing.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <button class="list-group-item item" @click="printPDF()">
+      <strong>Print Bill</strong>
+  </button>
+</template>
+
+<script>
+import  jsPDF from 'jspdf';
+export default {
+  props: ["items", "customer", "netTotal", "grandTotal"],
+  methods: {
+    printPDF() {
+      // console.log(this.items, this.customer, this.netTotal, this.grandTotal);
+
+      // get current date
+      let today = new Date();
+      let dd = today.getDate();
+      let mm = today.getMonth() + 1; //January is 0!
+      let yyyy = today.getFullYear();
+      if (dd < 10) {
+        dd = "0" + dd;
+      }
+      if (mm < 10) {
+        mm = "0" + mm;
+      }
+      today = mm + "/" + dd + "/" + yyyy;
+
+      // create new pdf object
+      let pdf = new jsPDF("p", "mm", [58, 100]);
```

Comment:
Make the template using HTML and then convert it to PDF. We already have a `frappe.getPDF` method. Just see how the PDF for Invoice is generated.